pkg/web_css/lib/src/_base.scss

```diff
@@ -393,31 +393,20 @@ pre {
   text-align: center;
 
   .dismisser {
-    float: right;
+    position: absolute;
```

Comment:
Not sure: does this need a `position: relative` in the parent element?

Reply:
I don't think so...